Samples/Desktop/D3D12Raytracing/src/D3D12RaytracingMiniEngineSample/ModelViewer.cpp

```diff
@@ -1051,8 +1096,25 @@ void D3D12RaytracingMiniEngineSample::Update( float deltaT )
       rayTracingMode = RTM_DIFFUSE_WITH_SHADOWRAYS;
     else if(GameInput::IsFirstPressed(GameInput::kKey_7))
       rayTracingMode = RTM_REFLECTIONS;
-
-    m_CameraController->Update(deltaT);
+	
+	static bool ignoreController = false;
+	
+	if (GameInput::IsFirstPressed(GameInput::kKey_left))
+	{
+		ignoreController = true;
+		m_CameraPosArrayCurrentPosition = (m_CameraPosArrayCurrentPosition + 1) % m_NumCameraPositions;
+	}
+	else if (GameInput::IsFirstPressed(GameInput::kKey_right))
```

Comment:
NIT: I'd add braces even though it's just a single statement for readability/consistency